brightener.h

```diff
@@ -6,7 +6,5 @@ class ImageBrightener {
 	std::shared_ptr<Image> m_inputImage;
```

Comment:
Any reason to retain a class with a member? `BrightenWholeImage` can be a function in a namespace, which accepts the image to be brightened... or is there any reason to keep the state in an object?